src/guide/extras/composition-api-faq.md

```diff
@@ -43,107 +43,107 @@ onMounted(() => {
 
 ### Can I use both APIs together?
 
-Yes. You can use Composition API via the [`setup()`](/api/composition-api-setup.html) option in an Options API component.
+Yes. You can use the Composition API via the [`setup()`](/api/composition-api-setup.html) option in an Options API component.
 
-However, we only recommend doing so if you have an existing Options API codebase that needs to integrate with new features / external libraries written with Composition API.
+However, we only recommend doing so if you have an existing Options API codebase that needs to integrate with new features / external libraries written with the Composition API.
 
-### Will Options API be deprecated?
+### Will the Options API be deprecated?
 
-No, we do not have any plan to do so. Options API is an integral part of Vue and the reason many developers love it. We also realize that many of the benefits of Composition API only manifest in larger-scale projects, and Options API remains a solid choice for many low-to-medium-complexity scenarios.
+No, we do not have any plan to do so. The Options API is an integral part of Vue and the reason many developers love it. We also realize that many of the benefits of the Composition API only manifest in larger-scale projects, and the Options API remains a solid choice for many low-to-medium-complexity scenarios.
 
-## Relationship with Class API
+## Relationship with the Class API
 
-We no longer recommend using Class API with Vue 3, given that Composition API provides great TypeScript integration with additional logic reuse and code organization benefits.
+We no longer recommend using the Class API with Vue 3, given that the Composition API provides great TypeScript integration with additional logic reuse and code organization benefits.
 
 ## Comparison with React Hooks
 
-Composition API provides the same level of logic composition capabilities as React Hooks, but with some important differences.
+The Composition API provides the same level of logic composition capabilities as React Hooks, but with some important differences.
 
-React Hooks are invoked repeatedly every time a component updates. This creates a number of caveats that can confuse even seasoned React developers. It also leads to performance optimization issues that can severely affect development experience. Here are some examples:
+React Hooks are invoked repeatedly every time a component updates. This creates a number of caveats that can confuse even seasoned React developers. It also leads to performance optimization issues that can severely affect the development experience. Here are some examples:
 
-- Hooks are call-order sensitive and cannot be conditional.
+- Hooks are call order sensitive and cannot be conditional.
```

Comment:
If we imagine that “call order” were spelled “callorder,” then a hyphen is appropriate if the qualifier precedes the noun, “hooks,” as in the fragment, “callorder-sensitive hooks.” It would be awkward to read, “call order-sensitive hooks” (though that would be technically correct). But luckily the qualifier does not preceed, but follows the noun, so the hyphon is not needed at all. I am sorry, I know I am not being very clear; so let us simply remember that “call order” is spelled without a hyphen.